### PR TITLE
Bradsdavis decompile filtering

### PR DESCRIPTION
**Pull request by jsight**
Continuation of https://github.com/windup/windup/pull/535

### CODE REVIEW

File: decompiler/impl-procyon/src/main/java/org/jboss/windup/decompiler/procyon/ProcyonDecompiler.java [hunk 1]
```diff
@@ -336,6 +326,21 @@ public DecompilationResult decompileArchive(final File archive, File outputDir,
                     jarEntryCount.decrementAndGet();
                     continue;
                 }
+                
+                //if filter exists, then filter...
```
Comment (lincolnthree) [inline decompiler/impl-procyon/src/main/java/org/jboss/windup/decompiler/procyon/ProcyonDecompiler.java]: This is a useless comment.

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/decompiler/ProcyonDecompilerOperation.java [hunk 2]
```diff
@@ -86,10 +88,12 @@ public void perform(final GraphRewrite event, final EvaluationContext context, f
                 outputDir = outputDir.toPath().resolve("WEB-INF").resolve("classes").toFile();
             }
 
+            // load all packages and filter on it.
```
Comment (lincolnthree) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/decompiler/ProcyonDecompilerOperation.java]: Also seems like a useless comment :)

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/65/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/694/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/66/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/695/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/67/
Test PASSed.

**Comment by jsight** [comment]
It is useful for debugging, and it really isn't logging so much as to be a performance issue, IMO.

**Comment by bradsdavis** [comment]
@jsight Useful for debugging, but not useful for info logging which is setup by default to the Windup log.  Let's just set it to be debugging (fine) grained if we have no other reason it should be info.

**Pull request merged**